### ISSUE TITLE
hide comments section entirely when it has no content

### ISSUE THREAD

**Issue by benjiwheeler** [LLK/scratch-www#2730]
if the project page's comments section has no content -- i.e., when commenting is turned off (or page is viewed in mobile browser width), there are no comments, and the project does not belong to the logged in user -- then we should probably hide the comments section entirely.

Also see related: https://github.com/LLK/scratch-www/issues/2590

**Comment by otsfan203** [comment]
Why?

**Comment by benjiwheeler** [comment]
@otsfan203 because if we show the section in that situation, it just says "Comments" and has no actual content, and that seems confusing.

**Comment by otsfan203** [comment]
@benjiwheeler Yes, but I don't think that should happen when commenting is turned off.

**Comment by carljbowman** [comment]
Also have some concerns about this. One chunk of work that we held off on due to timing was empty sections on the homepage (e.g. comments, studios). 

While there may not be content in these sections, we still want to show that they can have content. To me, these are great places to extend an inviting hand and/or give context:

_These are just sketches for sake of discussion. Language and design are not final and should be discussed further._

**Signed Out Example**
<img width="701" alt="screen shot 2019-02-05 at 8 52 52 am" src="https://user-images.githubusercontent.com/3409578/52277767-7a106600-2923-11e9-89d8-f6d86fe92ecb.png">

**Commenting Turned Off Example** 
<img width="716" alt="screen shot 2019-02-05 at 8 53 51 am" src="https://user-images.githubusercontent.com/3409578/52277813-94e2da80-2923-11e9-8947-d9adbda70879.png">

**Studio Example**
<img width="278" alt="screen shot 2019-02-05 at 8 54 21 am" src="https://user-images.githubusercontent.com/3409578/52277838-a4622380-2923-11e9-9cbf-4f3be9244a08.png">

**Comment by otsfan203** [comment]
@carljbowman I like those.

**Comment by otsfan203** [comment]
My suggestion would be to implement @carljbowman's suggestions, but when the project is unshared the "Comments" header and the turn off commenting option shouldn't show. When it has no content, (unless it is unshared) it should show so that users can comment.
